src/features/vault.ts

```diff
@@ -0,0 +1,173 @@
+import * as vscode from 'vscode';
+import * as utilAnsibleCfg from './utils/ansibleCfg';
+import * as tmp from 'tmp';
+import * as cp from 'child_process';
+import * as fs from "fs";
+
+async function askForVaultId(ansibleCfg: string) {
+  let vaultId: string = "default";
+  if (ansibleCfg === "ANSIBLE_VAULT_IDENTITY_LIST") {
+    let envVar : any = process.env.ANSIBLE_VAULT_IDENTITY_LIST;
+    var identityList = envVar.split(',').map( (id : string) => id.split('@', 2)[0].trim());
+  } else {
+    let cfg : any = utilAnsibleCfg.getValueByCfg(ansibleCfg);
+    var identityList = cfg.defaults.vaultIdentity_list.split(',').map( (id : string) => id.split('@', 2)[0].trim());
+  }
+  await vscode.window.showQuickPick(identityList)
+    .then( chosen => { if (chosen) { vaultId = chosen; } } );
+  return vaultId;
+}
+
+export var toggleEncrypt = async () => {
+	let editor = vscode.window.activeTextEditor;
+	if (!editor) {
+		return;
```

Comment:
I'm guessing we don't have lint set up and configured yet. The function above is using 2 spaces, I consider this the norm for JS and TS. The function is using a mix of 6 and 8 indent spaces.

Reply:
I have fixed all indents with c81640e6ad682f77bb6e4dbbba84743f593a43af